x-pack/plugins/apm/public/components/app/service_groups/service_groups_tour.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiSpacer, EuiText, EuiTourStep } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { ElasticDocsLink } from '../../shared/links/elastic_docs_link';
+
+export type TourType = 'createGroup' | 'editGroup' | 'serviceGroupCard';
+
+interface Props {
+  title: string;
+  content: string;
+  tourEnabled: boolean;
+  dismissTour: () => void;
+  children: React.ReactElement;
+}
+
+export function ServiceGroupsTour({
+  tourEnabled,
+  dismissTour,
+  title,
+  content,
+  children,
+}: Props) {
+  return (
+    <EuiTourStep
+      content={
+        <>
+          <EuiText size="s" color="subdued">
```

Comment:
```suggestion
          <EuiText size="s">
```
Just noticed this, but the main copy shouldn't be `subdued`